docs-source/src/architecture.rst

```diff
@@ -42,33 +42,484 @@ Thus, each license has a ``grace time`` period that is used to indicate how long
 After the ``grace time`` expires, the booking is deleted. This means that the license was checked out from the
 license server and doesn't need a booking anymore.
 
-License Manager Backend
------------------------
-The ``license-manager-backend`` provides a RESTful API where licenses, bookings and license configurations are tracked.
-The ``license-manager-agent`` uses this API to store the license usage information and to process the booking requests.
+License Manager API
+-------------------
+The ``License Manager API`` provides a RESTful API where licenses and bookins are tracked.
```

Comment:
Maybe cite that the CLI interacts with it as well?

Reply:
```suggestion
The ``License Manager API`` provides a RESTful API where licenses and bookins are tracked.
The ``License Manager Agent`` uses this API to store the license usage information and to process the booking requests.
The ``License Manager CLI`` interacts with this API to add new configurations and to check the usage information for each tracked license.
```

Better?

Reply:
Yes